code/backend/constants.py

```diff
@@ -0,0 +1,2 @@
+ERROR_429_MESSAGE = "We're currently experiencing a high number of requests for the service you're trying to access. Please wait a moment and try again.",
```

Comment:
Instead of having these constants in a separate file, can they live at the top of the `create_app.py`? Keeps the context together that way.

Reply:
Yeah, we can do that @adamdougal. Thought to maintain in a single place in case if we get more constants in future.

Reply:
Moved constants to the top of create_app.py